### PR TITLE
Julia 1.9 fixed - Update installation.md

### PR DESCRIPTION
**Pull request by PallHaraldsson**

### CODE REVIEW

**Comment by PallHaraldsson** [comment]
It's unclear to be it was stated 1.6 was fixed. I don't think that LTS version is. I wouldn't bother mentioning it, feel free. I believe 1.10 will be LTS and probably soon.

File: docs/src/installation.md [hunk 1]
```diff
@@ -63,12 +63,12 @@ The version of `libstdc++` shipped by Julia might be outdated if you are using a
     /home/user/julia-1.3.1/bin/../lib/julia/libstdc++.so.6: version `GLIBCXX_3.4.26' not found 
     (required by /home/user/R/x86_64-pc-linux-gnu-library/3.6/Rcpp/libs/Rcpp.so)
     
-Until this issue is fixed in Julia (see https://github.com/JuliaLang/julia/issues/34276) a workaround is to replace Julias `libstdc++` with the one of your OS:
+This iss was fixed in Julia 1.9 (see https://github.com/JuliaLang/julia/issues/34276) but a workaround for older Julia versions is to replace Julias `libstdc++` with the one of your OS:
```
Comment (palday) [inline docs/src/installation.md:66]:
```suggestion
This issue was fixed in Julia 1.9 (see https://github.com/JuliaLang/julia/issues/34276) but a workaround for older Julia versions is to replace Julia's `libstdc++` with the one of your OS:
```

**Comment by PallHaraldsson** [comment]
bump

**Review by palday — APPROVED** [review]
I'm unsure about this. Clearly there has been additional work in 1.9, but I also remember this mostly being a problem with Julia <= 1.5 -- I don't recall ever encountering this on 1.6+. I guess though the official close date of the linked issue was late 2022, so 1.9 is a reasonable cutoff point.

**Pull request merged**